scripts/app.js

```diff
@@ -25,32 +25,32 @@ Projects.all = [];
 Projects.prototype.toHtml = function (){
   let template = Handlebars.compile($('#project-template').text());
 
-  this.daysAgo = parseInt((new Date () - new Date(this.completedOn))/60/6-/24/1000);
+  this.daysAgo = parseInt((new Date () - new Date(this.completedOn))/60/60/24/1000);
   this.completeStatus = this.completedOn ? `completed ${this.daysAgo} days ago` : '(draft)';
 
   return template(this);
 };
 // added to start getting ready for json
-Project.loadAll = function(projectData) {
+Projects.loadAll = function(projectData) {
   projectData.sort(function(a,b) {
-    return (new Date(b.completedOn)) - new Date(a.completedOn));
+    return (new Date(b.completedOn) - new Date(a.completedOn));
   });
 
   projectData.forEach(function(ele){
-    Project.all.push(new Project(ele));
+    Projects.all.push(new Projects(ele));
   })
 }
 // added to start getting ready for json
-Project.fetchAll = function() {
-  if (localStorage = projectData) {
-    Project.loadAll(JSON.parse(localStorage.prprojectData));
+Projects.fetchAll = function() {
```

Comment:
Great job figuring out where your bugs are in regard to ``Project`` vs. ``Projects.``